apps/ios/Shared/Model/NtfManager.swift

```diff
@@ -108,6 +110,13 @@ class NtfManager: NSObject, UNUserNotificationCenterDelegate, ObservableObject {
                 actions: [],
                 intentIdentifiers: [],
                 hiddenPreviewsBodyPlaceholder: NSLocalizedString("New message", comment: "notifications")
+            ),
+            // TODO remove
+            UNNotificationCategory(
+                identifier: ntfCategoryMessageReceived,
```

Comment:
should it be a different category?
```suggestion
                identifier: ntfCategoryCheckingMessages,
```